### ISSUE TITLE
Also taking contributions from the Gerrit?

### ISSUE THREAD

**Issue by Dominator008** [tensorflow/tensorflow#1705]
In my opinion doing code review on GitHub is horrible except for very small fixes / tweaks. In addition, all those merge commits from "pull requests" really clutter up the git log.

I wonder if TensorFlow would consider also taking patches from Gerrit. Blaze (Bazel) does this: all external pull requests are not merged directly, but committed through the Gerrit repo. As a result, their commit history looks much cleaner: https://github.com/bazelbuild/bazel/commits/master.

**Comment by keveman** [comment]
@Dominator008, thanks for your comments. We did start out with maintaining both gerrit and github and accepting changes via both systems. However, the maintenance burden became too much, and with limited human resources we had (have), we decided to focus on making the  github process as robust as possible. Gerrit indeed has superior code review interface, but unfortunately, we are unable to dedicate resources to maintain multiple sources of truth for TensorFlow code.
